extensions/wikia/Email/Controller/FounderController.class.php

```diff
@@ -548,6 +551,14 @@ protected static function getEmailSpecificFormFields() {
 
 		return array_merge_recursive( $formFields, parent::getEmailSpecificFormFields() );
 	}
+
+	protected function findSignatureIcon() {
+		return ImageHelper::getFileUrl( 'Fandom-heart-color.png' );
+	}
+
+	protected function createEmailSignature() {
+		return $this->getMessage( 'emailext-emailconfirmation-community-team' )->text();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "emailext-emailconfirmation-community-team" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)